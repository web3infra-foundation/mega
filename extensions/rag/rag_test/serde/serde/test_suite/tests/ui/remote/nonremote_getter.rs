use serde_derive::Serialize;

#[derive(Serialize)]
struct S {
    #[serde(getter = "S::get")]
    a: u8,
}

impl S {
    fn get(&self) -> u8 {
        self.a
    }
}

fn main() {}
