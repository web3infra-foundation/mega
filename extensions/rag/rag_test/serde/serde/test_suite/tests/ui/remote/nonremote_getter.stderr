error: #[serde(getter = "...")] can only be used in structs that have #[serde(remote = "...")]
 --> tests/ui/remote/nonremote_getter.rs:4:1
  |
4 | / struct S {
5 | |     #[serde(getter = "S::get")]
6 | |     a: u8,
7 | | }
  | |_^
